.github/workflows/nano_unit_tests_pytorch.yml

```diff
@@ -200,9 +200,9 @@ jobs:
         pytorch-version: [
           # The reason why ipex1.10 is not currently supported is that the optimization of ipex1.10
           # will change the require_grad attribute of layers, which will make some tests of nano fail.
-          "torch==1.9.0+cpu torchvision==0.10.0+cpu torch_ipex==1.9.0",
+          "torch==1.9.0+cpu torchvision==0.10.0+cpu torch_ipex==1.9.0 intel_extension_for_pytorch==1.9.0",
```

Comment:
I think they changed the ipex's package name from torch_ipex to intel_extension_for_pytorch since 1.10.

![image](https://user-images.githubusercontent.com/10561966/197907755-ff3456d9-be4a-4ded-878c-c85412de9789.png)

Reply:
Thanks for this reminder!  Have revised workflow.